### PR TITLE
:rocket: Created navyfederal-7fh9xqpk

### PR DESCRIPTION
**Pull request by w00dbury**
This kit used Save Page WE to download the login page for Navy Federal Credit Union. I know the savepage-we IOK will also match on this page but I made this one to specifically match the target and the functions used to send credentials to Telegram. Let me know if multiple matches aren't allowed.

🟢Additions:
- Add [`navyfederal-7fh9xqpk`](6ebc8ddf601fde2659d796c179b29cd48c7302a2) to detect Navy Federal Credit Union phishing kit

### CODE REVIEW

**Review by IlluminatiFish — CHANGES_REQUESTED** [review]
Please consider the requested changes

File: indicators/navyfederal-7fh9xqpk.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+references:
+  - https://urlscan.io/result/0b9420f8-5aa2-442f-9e0c-ec3813aaea2a
+  - https://urlscan.io/result/65e0bb5b-76f0-45a1-a60f-af9971ea83ca
+
+related:
+  - id: savepage-we
+
+detection:
+  title:
+    html|contains: "<title>Navy Federal Credit Union - Our Members are the Mission®</title>"
```
[the remark this answers is not in the document]
Comment (w00dbury) [reply indicators/navyfederal-7fh9xqpk.yml]: @IlluminatiFish I didn't realize a title field existed, updated.

Are you thinking just a match on the exact date like [ce927f0](https://github.com/phish-report/IOK/pull/230/commits/ce927f055c07aa937d4b07fbd379b6f21c17e6c5)?

Thanks for your patience, I'm new to these IOKs but enjoy them so far.

[the remark this answers is not in the document]
Comment (w00dbury) [reply indicators/navyfederal-7fh9xqpk.yml]: Also I realized I used dom for the date here, if the value exists in the dom and the html is their a preference for using one over the other?

[the remark this answers is not in the document]
Comment (IlluminatiFish) [reply indicators/navyfederal-7fh9xqpk.yml]: > @IlluminatiFish I didn't realize a title field existed, updated.
> 
> Are you thinking just a match on the exact date like [ce927f0](https://github.com/phish-report/IOK/pull/230/commits/ce927f055c07aa937d4b07fbd379b6f21c17e6c5)?
> 
> Thanks for your patience, I'm new to these IOKs but enjoy them so far.

I think that should do it 👍

[the remark this answers is not in the document]
Comment (IlluminatiFish) [reply indicators/navyfederal-7fh9xqpk.yml]: > Also I realized I used dom for the date here, if the value exists in the dom and the html is their a preference for using one over the other?

As described in the field table, using the `html` field will only match on the HTML that is returned by the server, while the `dom` field will return the HTML after everything dynamic (e.g. JavaScript) has loaded and/or executed.

You may find it useful to use `dom` in instances where the website's content is dynamically loaded/generated. However I don't think it matters all too much if they value exists in both, for this rule at least.

**Review by IlluminatiFish — APPROVED** [review]
LGTM!

**Pull request merged**